pkg/archive/archive.go

```diff
@@ -18,7 +18,7 @@ import (
 	"syscall"
 
 	"github.com/Sirupsen/logrus"
-	"github.com/docker/docker/pkg/fileutils"
+	"github.com/docker/docker/builder/dockerignore"
```

Comment:
agreed. This archive package must not depend on docker internals. The `Excluder` struct should be moved to this package, without the whole `dockerignore` package.